docs/servers.md

```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
 
 To establish a dedicated server, a dedicated Linux or Windows machine is **highly** recommended.
 
-1. If you haven't already, install at least JRE and JDK 8.
-2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), or compile one yourself. 
-3. Open a terminal or TTY session then change `cd` to the directory the JAR is placed in.
-4. Run `java -jar server.jar` using Command Prompt (on Windows) or your favorite terminal (on Linux and Mac). The commands are explained in the `help` command.
-5. Start hosting a map with `host <mapname> [mode]` after you configured your server.
-6. If you are using Windows to run your server, use your favorite search engine to look up how to add rules to your Windows Firewall, as it blocks that port most of the time. Make sure to allow **port 6567 TCP and UDP**.
+1. If you haven't already, install Java 8. (If you can launch the client, this step is already done.)
+2. Download the desired server release from [itch.io](https://anuke.itch.io/mindustry), [Github releases](https://github.com/Anuken/Mindustry/releases). The filename should be `server-release.jar`, although if you are on Windows it may only show as `server-release`.
```

Comment:
It's not clear which version you want them to download here. The filename is `server.jar` on itch once extracted, not `server-release.jar`. The old guide only links to itch, so mentioning "server-release" and linking both here is confusing.

Reply:
Thanks, didn't notice that.